latch_cli/extras/nextflow/workflow.py

```diff
@@ -33,6 +37,9 @@
 
 
 def _get_flags_to_params(key: str, t: Type, flags: Dict[str, str]):
+    if is_output_dir(t):
```

Comment:
how does this work for parameters like `--outdir`

Reply:
oh wait we make those strings now right?

Reply:
Yeah those are strings now